app/api/server/api.js

```diff
@@ -659,12 +738,10 @@ settings.get('Accounts_CustomFields', (key, value) => {
 	}
 });
 
-settings.get('API_Enable_Rate_Limiter_Limit_Time_Default', (key, value) => {
-	defaultRateLimiterOptions.intervalTimeInMS = value;
-	API.v1.reloadRoutesToRefreshRateLimiter();
-});
-
-settings.get('API_Enable_Rate_Limiter_Limit_Calls_Default', (key, value) => {
-	defaultRateLimiterOptions.numRequestsAllowed = value;
-	API.v1.reloadRoutesToRefreshRateLimiter();
-});
+if (!process.env.TEST_MODE) {
+	settings.get(/^API_Rate_Limit_IP_.+/, () => API.v1.reloadRoutesToRefreshRateLimiter());
```

Comment:
Can we debounce the calls to `API.v1.reloadRoutesToRefreshRateLimiter`? To prevent refresh the routes multiple times while loading the settings?